src/components/common/textcopypopover/TextCopyPopover.tsx

```diff
@@ -22,46 +22,12 @@ interface TextCopyPopoverProps {
   maxWidth?: string,
```

Comment:
need a comma after className prop (may not be your code :) )

Reply:
Fixed